cmd/runner/main.go

```diff
@@ -388,13 +432,16 @@ func (h *handler) createAudit(ctx context.Context, taskID task.ID, req *task.Cre
 
 func (h *handler) createReport(ctx context.Context, taskID task.ID, req *task.CreateReportRequest) error {
 	fileNames := []string{}
-	for i, blobURI := range req.BlobURIs {
+	for _, blobURI := range req.BlobURIs {
 		// Load the parsed portfolio from blob storage, place it in /mnt/
 		// processed_portfolios, where the `create_report.R` script expects it
 		// to be.
-		fileName := fmt.Sprintf("%d.json", i)
-		fileNames = append(fileNames, fileName)
-		destPath := filepath.Join("/", "mnt", "processed_portfolios", fileName)
+		fileNameWithExt := filepath.Base(string(blobURI))
+		if !strings.HasSuffix(fileNameWithExt, ".json") {
+			return fmt.Errorf("given blob wasn't a JSON-formatted portfolio, %q", fileNameWithExt)
+		}
+		fileNames = append(fileNames, strings.TrimSuffix(fileNameWithExt, ".json"))
+		destPath := filepath.Join("/", "mnt", "processed_portfolios", fileNameWithExt)
```

Comment:
mnt?

Reply:
https://tldp.org/LDP/Linux-Filesystem-Hierarchy/html/mnt.html

Basically, `/mnt` is a generic place for external things being mounted in. [It's a convention Alex used in the stub repo](https://github.com/AlexAxthelm/stub_PACTA_containers_for_app/blob/190763cba67bafdf666f56eb34693da728fea21b/combined/create_report.R#L13), and it makes as much sense as any other location.

Reply:
:+1: